### ISSUE TITLE
Mac: Errors when installing pip libraries via setup

### ISSUE THREAD

**Issue by mwobensmith** [mozilla/iris#72]
When invoking `pip install -e . `to run the `setup.py` file, we get an error:

`AssertionError: You must first install pyobjc-core and pyobjc: https://pyautogui.readthedocs.io/en/latest/install.html `

However, we have already added `pyobjc` and `pyobjc-core ` in the setup file, so it should be available.

The workaround is to install `pyobjc` and `pyobjc-core ` individually before running `pip install -e .`, but we'd like to not have to do this.

**Comment by mwobensmith** [comment]
Note: remove commented lines in `setup.py` when complete.

**Comment by mwobensmith** [comment]
Current workaround is to install these two packages via the bootstrap script, but will investigate a cleaner fix.

**Comment by mwobensmith** [comment]
I ran into a problem on Mac. When I run `pip install -e .` it will show errors when installing `pyobjc`.

It gets pretty far, and this is the last part that is successful:
```
Building wheels for collected packages: PyAutoGUI
  Running setup.py bdist_wheel for PyAutoGUI ... done
  Stored in directory: /Users/mwobensmith/Library/Caches/pip/wheels/33/9d/3d/88afd520024d2213a6ec7d9c8e9d5108b0259156f1fb86c071
Successfully built PyAutoGUI
```
Then, I get a lot of errors, starting with this:

```
pyobjc-framework-mapkit 5.0 has requirement pyobjc-core>=5.0, but you'll have pyobjc-core 4.2.2 which is incompatible.
pyobjc-framework-mapkit 5.0 has requirement pyobjc-framework-Quartz>=5.0, but you'll have pyobjc-framework-quartz 4.2.2 which is incompatible.
pyobjc-framework-applicationservices 5.0 has requirement pyobjc-core>=5.0, but you'll have pyobjc-core 4.2.2 which is incompatible.
pyobjc-framework-applicationservices 5.0 has requirement pyobjc-framework-Quartz>=5.0, but you'll have pyobjc-framework-quartz 4.2.2 which is incompatible.
pyobjc-framework-externalaccessory 5.0 has requirement pyobjc-core>=5.0, but you'll have pyobjc-core 4.2.2 which is incompatible.
pyobjc-framework-syncservices 5.0 has requirement pyobjc-core>=5.0, but you'll have pyobjc-core 4.2.2 which is incompatible.
pyobjc-framework-ituneslibrary 5.0 has requirement pyobjc-core>=5.0, but you'll have pyobjc-core 4.2.2 which is incompatible.
pyobjc-framework-medialibrary 5.0 has requirement pyobjc-core>=5.0, but you'll have pyobjc-core 4.2.2 which is incompatible.
pyobjc-framework-medialibrary 5.0 has requirement pyobjc-framework-Quartz>=5.0, but you'll have pyobjc-framework-quartz 4.2.2 which is incompatible.
pyobjc-framework-gamekit 5.0 has requirement pyobjc-core>=5.0, but you'll have pyobjc-core 4.2.2 which is incompatible.
pyobjc-framework-gamekit 5.0 has requirement pyobjc-framework-Quartz>=5.0, but you'll have pyobjc-framework-quartz 4.2.2 which is incompatible
```
When I invoke Iris, I get this:
```
Traceback (most recent call last):
  File "/Users/mwobensmith/PycharmProjects/iris/bin/iris", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/Users/mwobensmith/PycharmProjects/iris/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3112, in <module>
    @_call_aside
  File "/Users/mwobensmith/PycharmProjects/iris/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3096, in _call_aside
    f(*args, **kwargs)
  File "/Users/mwobensmith/PycharmProjects/iris/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3125, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/Users/mwobensmith/PycharmProjects/iris/lib/python2.7/site-packages/pkg_resources/__init__.py", line 580, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/Users/mwobensmith/PycharmProjects/iris/lib/python2.7/site-packages/pkg_resources/__init__.py", line 593, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/Users/mwobensmith/PycharmProjects/iris/lib/python2.7/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (pyobjc-framework-Quartz 4.2.2 (/Users/mwobensmith/PycharmProjects/iris/lib/python2.7/site-packages), Requirement.parse('pyobjc-framework-Quartz==5.0'), set(['pyobjc']))
```

**Comment by mwobensmith** [comment]
Never mind. I was able to fix just by specifying version info for the pyobjc stuff. It all works now. 

```
    'pyobjc-core==5.0;platform_system=="Darwin"',
    'pyobjc==5.0;platform_system=="Darwin"',
    'pyobjc-framework-Quartz==5.0;platform_system=="Darwin"',
```

**Comment by mwobensmith** [comment]
Fixed with fix for #72
